### PR TITLE
Support patch parameters

### PR DESCRIPTION
**Pull request by irdkwia**
- AddExperienceShare has a parameter to set the % of exp. shared
- ChangeTextBoxBackground has 4 parameters to directly set the textbox background color
- ChangeMoveStatsDisplay has an option to add the power/accuracy value next to the bar. It also has a parameter to set the start ID of used text icons in markfont.dat, which allows to move the new icons anywhere in the file. Note that it does not check if selected entries are empty, so make sure that the selected entries can be used before applying the patch.

### CODE REVIEW

File: skytemple_files/patch/handler/move_growth.py [hunk 1]
```diff
@@ -67,15 +64,15 @@
                5000,1,1,1,
                   0,5,5,1]
 
-class MoveGrowthPatchHandler(AbstractPatchHandler):
+class MoveGrowthPatchHandler(AbstractPatchHandler, DependantPatch):
```
Comment (Parakoopa) [inline skytemple_files/patch/handler/move_growth.py:67]: i think this is a mistake?

File: skytemple_files/patch/patches.py [hunk 2]
```diff
@@ -92,7 +92,7 @@ class PatchType(Enum):
     EXTRACT_MOVE_CODE = ExtractMoveCodePatchHandler
     EXTRACT_ITEM_CODE = ExtractItemCodePatchHandler
     EXTRACT_SP_CODE = ExtractSPCodePatchHandler
-    MOVE_GROWTH = MoveGrowthPatchHandler
+    #MOVE_GROWTH = MoveGrowthPatchHandler
```
Comment (Parakoopa) [inline skytemple_files/patch/patches.py:95]: and this probably too?

**Review by Parakoopa — CHANGES_REQUESTED** [review]

File: skytemple_files/patch/handler/move_growth.py (hunk 1 above)
Comment (irdkwia) [reply skytemple_files/patch/handler/move_growth.py:67]: What's the problem?

Comment (Parakoopa) [reply skytemple_files/patch/handler/move_growth.py:67]: Oh nevermind, I didn't see the `depends_on` that was already there in the review screen.

File: skytemple_files/patch/patches.py (hunk 2 above)
Comment (irdkwia) [reply skytemple_files/patch/patches.py:95]: I removed it from the list because it depends on a patch that won't be merged until 1.4.

**Comment by Parakoopa** [comment]
I merged the other PR, can you re-enable the growth patch?

**Review by Parakoopa — APPROVED** [review]

**Pull request merged**